backend/core/interactem/core/models/messages.py

```diff
@@ -31,7 +31,7 @@ class MessageSubject(str, Enum):
 
 class MessageHeader(BaseModel):
     subject: MessageSubject
-    meta: dict[str, Any] = {}
+    meta: bytes | dict[str, Any] = b"{}"
```

Comment:
Given that using a pydantic model for `meta` is going to lead to poor performance, should we just change the type of `meta` to be `msgspec.Struct`, so that we could bring the logic for serialization and deserialization into interfaceEM, we would need away for operators to register their meta types. Or maybe even the whole header should be a Struct. Thoughts?